tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/2/out~MOOSE_parser.csv

```diff
@@ -0,0 +1,2 @@
+time,average_fuel_temperature,average_graphite_temperature,delta_temp,total_heat_source
+0,0,0,0,0
```

Comment:
I do not recall. Is this file generated by the Code Interface or by MOOSE itself? (In the first case, it should be removed)

Reply:
I agree the out~MOOSE_parser should be removed, but isn't out~MOOSE_parser.csv one of the output files that the interface check scoops up and checks collecting output for?

Reply:
Sorry, I forgot.
Yes the CSVs should stay.

Reply:
Yep, the interface check needs it to pick up stuff after it *pretends* to run the executable. I left it in the amended commit.